frontends/mit-open/src/pages/FieldPage/FieldPageSkeleton.tsx

```diff
@@ -61,6 +71,25 @@ const FieldSkeletonProps: React.FC<FieldSkeletonProps> = ({
   const field = useChannelDetail(String(channelType), String(name))
   const urlParams = new URLSearchParams(field.data?.search_filter)
   const displayConfiguration = field.data?.configuration
+  const memoizedUrlParams = useMemo(() => {
+    const urlParams = new URLSearchParams(field.data?.search_filter)
+    return urlParams
+  }, [field.data?.search_filter])
+
+  const urlParamMap: Record<string, string[] | string> = useMemo(() => {
+    return getSearchParamMap(memoizedUrlParams)
+  }, [memoizedUrlParams])
+
+  const FEATURED_RESOURCES_CAROUSEL: ResourceCarouselProps["config"] = [
+    {
+      cardProps: { size: "small" },
```

Comment:
On figma, it looks like medium cards to me? 4 are shown, right?
<img width="566" alt="Screenshot 2024-06-11 at 2 26 12 PM" src="https://github.com/mitodl/mit-open/assets/9010790/45f3a510-d16a-4677-b3b0-8eaaccc64c18">

Reply:
pushed up the sizing changes